.github/workflows/build_and_test.yml

```diff
@@ -11256,3 +11291,40 @@ jobs:
     - uses: ammaraskar/gcc-problem-matcher@0.1
     - name: Build
       run: cd OpenDDS && docker build -f .github/workflows/build_u14_gcc4.Dockerfile .
+
+  openssl3_u20:
+
+    runs-on: ubuntu-20.04
+
+    steps:
+    - name: checkout openssl 3.0.2
+      run: |
+        git clone --branch openssl-3.0.2 --depth 1 https://github.com/openssl/openssl.git
+        cd openssl
+        echo "OPENSSL_COMMIT=$(git rev-parse HEAD)" >> $GITHUB_ENV
+    - name: Cache Artifact
+      id: cache-artifact
+      uses: actions/cache@v2
+      with:
+        path: ${{ github.job }}.tar.xz
+        key: c01_${{ github.job }}_${{ env.OPENSSL_COMMIT }}
+    - name: openssl 3.0.2
+      if: steps.cache-artifact.outputs.cache-hit != 'true'
+      run: |
+        PREFIX=$PWD/openssl_install
+        echo PREFIX=$PREFIX
+        mkdir $PREFIX
+        cd openssl
+        ./Configure no-tests --prefix=$PREFIX
+        make install_sw
```

Comment:
Would a `-j` option help speed this up?

Reply:
Was this the cause of...
```
mv: cannot stat 'apps/lib/libapps-lib-app_libctx.d.tmp': No such file or directory
make[1]: *** [Makefile:3857: apps/lib/libapps-lib-app_libctx.o] Error 1
```
https://github.com/objectcomputing/OpenDDS/runs/5975598293?check_suite_focus=true

Maybe the makefiles from OpenSSL don't work with parallel builds